lib/oli_web/live_session_plugs/set_user.ex

```diff
@@ -28,23 +28,25 @@ defmodule OliWeb.LiveSessionPlugs.SetUser do
     case AccountLookupCache.get_author(current_author_id) do
       {:ok, current_author} ->
         case current_author do
-          %Author{system_role_id: system_role_id} ->
+          %Author{} ->
             socket
             |> assign(
               current_author: current_author,
-              is_system_admin: system_role_id == Oli.Accounts.has_admin_role?(current_author)
+              is_system_admin: Oli.Accounts.is_system_admin?(current_author),
```

Comment:
This looks good for now, eventually we will probably want to update where we are checking this
`is_system_admin` to instead use more of a role-based access control instead of this flag